### PR TITLE
DEVP-3487 updated deps, removed deprecations and re-gen the code

### PR DESCRIPTION
**Pull request by MickStanciu**
## Summary

* Updated the go.mod because of a few security vulnerabilities
* Removed the deprecation
* Regenerated the GO Files

## Problem description

Keeping things up2 date

## Checklist

- [ ] Is this PR a reasonable size?

- [ ] List deployment sequence with all relevant PRs.
  - ...

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
+	google.golang.org/protobuf v1.34.2
 )
 
 require (
 	github.com/corpix/uarand v0.1.1 // indirect
-	golang.org/x/text v0.4.0
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/stretchr/testify v1.7.0 // indirect
+	golang.org/x/text v0.18.0
```
Comment (lachlancooper) [inline go.mod:17]: Can we move this to the direct block above?

Comment (MickStanciu) [reply go.mod:17]: sure

**Review by lachlancooper — APPROVED** [review]
